BitMex,BitStamp: change YES to WIP

knocte opened the pull request:
These exchanges, even if they have signaled support (yellow circle), they **still** don't support LN, and it's not fair to give them a YES.

theDavidCoen commented:
Hi @knocte ,
thanks for the PR but the status is already correct.
If you look at the legend:
"🟡 Signaling support (example: the exchange has LN nodes, will fully support LN but at the moment doesn't support deposits/withdrawals)"

knocte commented:
I have read the legend many times. IMHO a status like that still doesn't deserve a big uppercase YES word in the "Lightning Support" column, think about it. Especially since we're talking about BitMex and BitStamp here who "supposedly" signaled support for LN.. when? in JUL2019 and AUG2018, so around 5-6 years ago. Can we really keep giving them such a seal of approval really?

knocte commented:
Even a WIP word would be generous! We're talking about 5-6 of "in progress" development.

theDavidCoen commented:
Yet they have Lightning support if they have nodes and provide liquidity.
However, we can modify the legend so ⚡ means they provide full support for deposit and withdrawal, ⏳ means Work in Progress and 🟡  means just signaling support / they have nodes but they miss deposit or withdrawal or both.

knocte commented:
> we can modify the legend so ⚡ means they provide full support for deposit and withdrawal, ⏳ means Work in Progress and 🟡 means just signaling support / they have nodes but they miss deposit or withdrawal or both.

Did you notice that the legend already says this? In fact I'm not changing any icon in this PR, but a different column. A column that is actually not covered by the legend.

theDavidCoen commented:
I see no additional columns in your changes.

knocte commented:
I didn't say that I added an additional column, I said that the things I changed are in a column whose content is *not* covered by the legend. The legend is just explaining the icons in the status column and I didn't modify the status column at all.

theDavidCoen commented:
I see, but I don't want to introduce a new option in "Lightning Support?" column other then YES or NO. 
I prefer to have a more explanatory legend or an additional symbol in Status.
I'm also open for a completely new column such as "Lightning deposit / withdrawal?" near "Lightning Support?" .

knocte commented:
> column other then YES or NO.

Then strictly speaking Bitstamp and BitMex should have a *NO*, not a YES.
Is it that you don't like the word WIP? how about an ellipsis (...)?

> I'm also open for a completely new column such as "Lightning deposit / withdrawal?" near "Lightning Support?" .

Why would it matter for a user that an exchange has LN "support" without supporting withdrawals or deposits? LN support can only be enabled by enabling withdrawals and deposits!

theDavidCoen commented:
Supporting Lightning Network in this repo doesn't strictly mean to support deposits/withdrawals.
Just like it doesn't mean that the exchange has to support routing.

The Status indicates the level of support and that can be improved of course.
But in Support Lightning? I want to keep a binary choice.

knocte commented:
Then you should rename the "Lightning Support" column to "Lightning Advocacy".

theDavidCoen commented:
I'm closing this one since solved by #65 and Legend update.
